style.css

```diff
@@ -66,11 +66,48 @@ body {
 }
 
 .menu__item {
-    padding: 0 13px;
+    position: relative;
+    margin-right: 26px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6252582)